near-sdk/src/near_annotations.rs

```diff
@@ -0,0 +1,25 @@
+//! `#[near]` and `#[near_bindgen]` documentation module
+//!
+//! This is not a real module; here we document the attributes that [`#[near]`](crate::near)
+//! and [`#[near_bindgen]`](crate::near_bindgen) macro use.
```

Comment:
This should document which sub-attributes are specific to only `#[near]` , and which are shared by both. 
`#[near]`-specific ones must be `serializers` and `contract_state`, cannot remember anything else

Reply:
or better even under each individual attribute, `serializers` and `contract_state`, that they are near-specific, not supported by near_bindgen.

Reply:
Should document under each attribute, not in index in one of possible paths to reach this attribute